src/functions.js

```diff
@@ -378,6 +371,19 @@ defineFunction([
     };
 });
 
+// \mathop class command
+defineFunction("\\mathop", {
+    numArgs: 1,
+}, function(context, args) {
+    var body = args[0];
+    return {
+        type: "op",
+        limits: false,
+        symbol: false,
```

Comment:
`symbol` is linked with the argument type of `value`. if `symbol` is true, then `value` must be suitable for passing to `makeSymbol`, which isn't true here.